docs/content/concepts/flow-concepts/slashing.mdx

```diff
@@ -68,11 +68,11 @@ Transitioning to a rigorous staking protocol in which all slashable conditions a
 
 **Adjudication:** Consensus nodes evaluate the content of the collection, if the collection is found invalid, the Collection Node who signed the content is slashed.
 
-**1.3 Double Collection Proposal Witness (DCPW):** Collection nodes of a cluster run a mini consensus inside the cluster to decide on a collection, which requires collectors to propose the collection and aggregate votes from others. During the collection consensus, if a Collection Node proposes more than one proposal, any other Collection Node inside the cluster can report a Double Collection Proposal Witness (including both proposals).
+**1.3 Double Collection Proposal Witness (DCPW):** Collection nodes of a cluster run a mini consensus inside the cluster to decide on a collection, which requires Collection nodes to propose the collection and aggregate votes from others. During the collection consensus, if a Collection Node proposes more than one proposal, any other Collection Node inside the cluster can report a Double Collection Proposal Witness (including both proposals).
```

Comment:
```suggestion
**1.3 Double Collection Proposal Witness (DCPW):** Collection nodes of a cluster run a mini consensus inside the cluster to decide on a collection, which requires collection nodes to propose the collection and aggregate votes from others. During the collection consensus, if a collection node proposes more than one proposal, any other collection node inside the cluster can report a Double Collection Proposal Witness (including both proposals).
```

Reply:
lower case node type ? as we discussed on the slack thread earlier

Reply:
I think we'll wait, because there are multiple instances of the previous casing.